### PR TITLE
Make info categories team controlled

### PR DESCRIPTION
**Pull request by valberg**
Currently the info categories are maintained by the orga team, this is highly inefficient.

We should hand the responsibility over to the teams that are responsible for, and know more about, their own area.

- [x] Use [django-reversion](https://django-reversion.readthedocs.io/) to make `InfoItem` version controlled
- [x] Build a way to maintain an `InfoCategory` as a team responsible
  - [x] Create `InfoItem`
  - [x] Edit `InfoItem`
  - [x] Delete `InfoItem`
- [x] Add a "quick link" on an `InfoItem` on the info page itself when a team responsible is logged in.
- [x] Show what team is responsible for a given `InfoCategory`

### CODE REVIEW

File: src/info/models.py [hunk 1]
```diff
@@ -29,15 +31,33 @@ class Meta:
...
+
     def clean(self):
         if InfoItem.objects.filter(category__camp=self.camp, anchor=self.anchor).exists():
             # this anchor is already in use on an item, so it cannot be used (must be unique on the page)
-            raise ValidationError({'anchor': 'Anchor is already in use on an info item for this camp'})
+            raise ValidationError(
+                {'anchor': 'Anchor is already in use on an info item for this camp'}
+            )
+
+        if self.team.camp != self.camp:
```
Comment (tykling) [inline src/info/models.py]: is this check needed?

Comment (valberg) [reply src/info/models.py]: Just to make sure we don't accidentally add a 2017 team to be responsible for a 2018 info category :)

Comment (tykling) [reply src/info/models.py]: the category FK to camp should be deleted and replaced with a camp property that returns team.camp

**Comment by tykling** [comment]
can you resolve the conflicts and I will get this merged

**Comment by valberg** [comment]
@tykling Done!

**Pull request merged**